atom/common/extensions/api/BUILD.gn

```diff
@@ -0,0 +1,39 @@
+# Copyright 2015 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# import("//build/config/features.gni")
+# import("//build/config/ui.gni")
+# import("//chrome/common/extensions/api/api_sources.gni")
+# import("//chrome/common/features.gni")
```

Comment:
are these needed? if not, remove

Reply:
The contents of this file was copied from https://cs.chromium.org/chromium/src/chrome/common/extensions/api/BUILD.gn

These imports weren't needed and can be removed.